os.go

```diff
@@ -111,3 +120,15 @@ func (OsFs) SymlinkIfPossible(oldname, newname string) error {
 func (OsFs) ReadlinkIfPossible(name string) (string, error) {
 	return os.Readlink(name)
 }
+
+func (OsFs) ReadFile(name string) ([]byte, error) {
+	return os.ReadFile(name)
+}
+
+func (OsFs) ReadDir(dirname string) ([]fs.DirEntry, error) {
```

Comment:
Maybe add a pair of files, one for go versions before io/fs existed, defining DirEntry, and a second for newer versions that defines it as an alias?

Reply:
Yes, I saw that was done for a few other libraries after I submitted the patch :)